apps/patient/src/components/BrandedPharmacyCard.tsx

```diff
@@ -51,40 +49,24 @@ export const BrandedPharmacyCard = ({ pharmacyId, selectedId, handleSelect }: Pr
   const brand = PHARMACY_BRANDING[pharmacyId];
   if (!brand) return null;
 
-  let tagline = null;
-  if (pharmacyId in capsulePharmacyIdLookup) {
-    const [firstWord, ...restOfSentence] = brand.description.split(' ');
-
-    tagline = (
-      <Text fontSize="sm" display="inline">
-        <Box as="span" color="gray.900" mr={1}>
-          {firstWord}
-        </Box>
-        <Box as="span" color="gray.500">
-          {restOfSentence.join(' ')}
-        </Box>
-      </Text>
-    );
-  } else {
-    tagline = (
-      <Text fontSize="sm" color="gray.500">
-        {brand.description}
-      </Text>
-    );
-  }
+  const tagline = brand.description ? (
+    <Text fontSize="sm" color="gray.500">
+      {brand.description}
+    </Text>
+  ) : null;
 
   return (
     <SlideFade offsetY="60px" in={true} key={`courier-pharmacy-${pharmacyId}`}>
       <Card
         bgColor="white"
         cursor="pointer"
-        onClick={() => handleSelect(pharmacyId)}
+        onClick={() => handleSelect && handleSelect(pharmacyId)}
```

Comment:
can also use optional chaining here 
`onClick={() => handleSelect?.(pharmacyId)}`

Reply:
I had the same thing here lol